src/components/graph/graph.builder.js

```diff
@@ -189,6 +198,8 @@ function buildNodeProps(node, config, nodeCallbacks = {}, highlightedNode, highl
         opacity,
         renderLabel: config.node.renderLabel,
         size: nodeSize * t,
+        width: nodeSizeWidth,
```

Comment:
> The `size` property above yours is been multiplied by a transform value, the variable `t`. You probably should apply the same transformation to both `width` and `height`, so that transformation is preserved in rectangular nodes as well. 👌

Reply:
fixed

Reply:
I changes this so the width and height are multiplied by t but now when you have a rectangle the shape of the rectangle changes as you zoom in and out.  I don't think we want that.  The same would go for the square shapes I don't think we want them to resize based on the zoom level because then we are really applying the zoom twice.
I'm suggesting remove the *t from size,width and height.